src/svgp.jl

```diff
@@ -17,55 +20,60 @@ which can be found in closed form.
 variational Gaussian process classification." Artificial Intelligence and
 Statistics. PMLR, 2015.
 """
-function AbstractGPs.approx_posterior(::SVGP, fz::FiniteGP, q::AbstractMvNormal)
+function AbstractGPs.posterior(svgp::SVGP)
+    q, fz = svgp.q, svgp.fz
     m, A = mean(q), _chol_cov(q)
```

Comment:
In the notebooks you use `A` to denote the root of cov(q), but the `Cholesky` object returned by `_chol_cov` represents the actual covariance - maybe rename it to
```suggestion
    m, S = mean(q), _chol_cov(q)
```
or something like that to make that clearer?